test/smoke_test.dart

```diff
@@ -9,7 +9,6 @@ void main() {
     final application = Application();
     await tester.pumpWidget(application);
     expect(find.byType(GoogleMap), findsOneWidget);
-    expect(find.byType(FloatingActionButton), findsNWidgets(2));
```

Comment:
How come you deleted this? Can't you just change the number of widgets to be found to match what you now have? =3

Reply:
0 but whats the point